Tutorials/Level2Tutorial.md

```diff
@@ -0,0 +1,1081 @@
+during graph construction, thus placing a large burden on rank-0.  When 
+using the parallel graph loader, users have the ability to split a simulation 
+graph into `N` files for `N` ranks of a simulation whereby each `N-th` rank
+independently loads only its portion of the simulation graph.  SST will handle 
+the hard portion of wiring up the graph across MPI ranks.  However, the input 
+files for each rank *must* contain the same `program_options`.  In this manner, 
+each rank is configured to operate under the same global options.
+
+Executing SST with the parallel graph loader requires that the user create 
+a single input file for each rank of the parallel simulation.  As a result, 
+if the user seeks to utilize `4` ranks, then `4` input files must be present.  
+The files are named using a monotonically increasing integer in the following 
+form (for `n` ranks):
+```
+FILE0.json
+FILE1.json
+...
+FILE{n-1}.json
+```
+
+When executing using the parallel graph loader, we need additional options 
+and file prefixes on the command line.  First, we must specify the 
+`--parallel-output` option in order to trigger the SST core to load 
+the per-rank simulation files.  Next, we must specify the base file 
+prefix as the input to the simulation.  Using our example above (`FILE0.json`) 
+would be specified as `FILE.json`.  An example of doing so resembles the following:
+
+```
+$> mpirun --hostfile output.txt -n2 sst --parallel-output=1 FILE.json
+```
+
+In addition to the aforementioned parallel graph loading, the latest development 
+source for SST also supports converting large simulation graphs into their 
+parallel form.  This allows users to easily port their existing Python-based 
+simulation inputs to parallel JSON inputs.  Users may accomplish this similar to the 
+basic JSON convertor outlined above by adding the `--output-json=FILE.json` 
+to the parallel run.  Note that the file name specified in the `--output-json` flag 
+must contain a `.json` file extension.  SST will automatically split the input 
+Python file into the appropriate `N` output JSON files in the same monotonically 
+increasing form as outlined above.
+
+```
+$> mpirun --hostfile output.txt -n2 sst --parallel-output=1 --output-json=FILE.json FILE.py
+```
+
+The following are several candidate examples of parallel JSON input files using 
+standard SST tests.
+
+|  **Description**  | **Ranks** | **Rank Files** |
+|:-|:-:|:-|
+| Miranda Stencil3D | 2 | [Rank0](samples/parallel/stencil3dbench0.json) [Rank1](samples/parallel/stencil3dbench1.json)|
+| Merlin 256 Node FatTree | 4 | [Rank0](samples/parallel/fattree_256_test_parallel0.json) [Rank1](samples/parallel/fattree_256_test_parallel1.json) [Rank2](samples/parallel/fattree_256_test_parallel2.json) [Rank3](samples/parallel/fattree_256_test_parallel3.json)|
+
+## Links to External SST Components: <a name="ExternSSTComp"></a>
+* [HBM DRAMSIm2](https://github.com/tactcomplabs/HBM/releases/tag/sst-8.0.0-release)
```

Comment:
Is this section just a link to what's available? Can you instead point users to check: http://sst-simulator.org/SSTPages/SSTMainDocumentation/ and click on "External Components Instructions" (the actual page URL has is linked to a specific release)? This has a complete list as well as links to the versions/tags we test against. There's also a page that links to community elements that we don't necessarily test against. (http://sst-simulator.org/SSTPages/SSTMainCommunity/)

Reply:
What?  No selfish plug?  :-)
Fixed in the branch